### PR TITLE
NEP: Regenerate table in NEP 29 (add numpy 1.18 and 1.19 to list)

### PR DESCRIPTION
**Pull request by Carreau**
Numpy 1.18 and 1.19 have been released, which add two new recommended
stop date for recommended numpy support in 2021 and 2022.

### CODE REVIEW

File: doc/neps/nep-0029-deprecation_policy.rst [hunk 1]
```diff
@@ -111,7 +111,9 @@ Jun 23, 2020 3.7+   1.15+
 Jul 23, 2020 3.7+   1.16+
 Jan 13, 2021 3.7+   1.17+
 Jul 26, 2021 3.7+   1.18+
-Dec 26, 2021 3.8+   1.18+
+Dec 22, 2021 3.7+   1.19+
+Dec 26, 2021 3.8+   1.19+
+Jun 21, 2022 3.8+   1.18+
```
Comment (rgommers) [inline doc/neps/nep-0029-deprecation_policy.rst:116]: This and the next line should be `1.20+` right?

**Review by rgommers — APPROVED** [review]
Thanks @Carreau. Update LGTM modulo a small copy-paste issue.

File: doc/neps/nep-0029-deprecation_policy.rst (hunk 1 above)
Comment (Carreau) [reply doc/neps/nep-0029-deprecation_policy.rst:116]: Hum, there is likely a bug in the Python at the bottom of the NEP;  I blindly copy pasted. Will look into it:

Screen Shot 2020-09-17 at 08 57 04

**Comment by Carreau** [comment]
Fixed the Python snippet. 

I'm thinking on adding a `*` to the versions number that have not been released yet, like:

```
Jun 21, 2022 3.8+   1.20*
Apr 14, 2023 3.9*   1.20*

*: Those versions have not been released yet and schedule or version numbering may change in the future.
```

Thoughts ? This can also be done in a subsequent PR.

**Comment by rgommers** [comment]
> I'm thinking on adding a `*` to the versions number that have not been released yet, like:

That sounds like a good idea to me.

**Comment by rgommers** [comment]
Let's get this in, follow-up for `*` very welcome.

**Pull request merged**